test/endpoints/automations.test.js

```diff
@@ -121,9 +261,9 @@ describe('Tests for automation endpoints\n', () => {
         expect(res.body)
           .to.have.property('message')
           .to.equal('Successfully fetched automations');
-        expect(res.body.pagination)
-          .to.have.property('currentPage')
-          .to.be.equal(1);
+        expect(res.body.data[0])
+          .to.have.property('type')
```

Comment:
using the generated automation in the before script get all the automation of type onboarding and offboarding